### DISCUSSION TITLE
Mic -> GarageBand -> BH -> Google Meet

### DISCUSSION THREAD

**Discussion by fabiosz** [ExistentialAudio/BlackHole#112] [General]
Hi, I'm trying to set up BlackHole to use it as a means to route sound from my external USB mic to GarageBand (so I can use a noise gate) and re-route the signal from GarageBand to Apps like Skype or Google Meet. As of now I'm not able to see any signal from Skype when I select either Blackhole 16ch or Aggregate for the input source.
I have tried setting up an Aggregate device in Audio Devices on my Mac, which includes Blackhole 16ch, my RODE mic both "in" and "out", Built-in Microphone and Built-in Output. Then in GarageBand I select RODE as an input and Blackhole as an output. But when I try to select Aggregate or Blackhole on Skype as an input device, people cannot hear my voice. Can you help me? 

Also, I'd need to be able to listen to the audio coming from the call, either from the computer output or the microphone monitor headphones jack output. How should I proceed? 

Thanks in advance for your kind help.

**Comment by devinroth** [comment]
Do you need to hear your voice too?

**Comment by fabiosz** [comment]
Not an essential part but it'd be useful I guess.

**Comment by devinroth** [comment]
Let's try without first. Easier method.
1. Set the aggregate as input and output in GB.
2. In an audio track set the input to your mic and the output to BH.
3. Set BH as mic in Google Meet. You can set your speaker to the device you want to hear audio from.

**Comment by fabiosz** [comment]
Thank you Devin. Unfortunately, I don't seem to be able to set an output for a track in GB. Do you by any chance know how to do that? I can set input and output in the GB Preferences, which I did. But I don't find track output settings anywhere.

**Comment by fabiosz** [comment]
Ok, it seems though that I can still select the source in Skype and Skype actually gets what's coming out of GB. For some reason however, I hear double audio feedback (it's like direct feedback + very low latency feedback) in my headphones. My headphones are connected to the Mac and in the system prefs I have selected RODE USB for both I and O. Should I do differently? Thanks in advance for your help.
